agent/api/containerstatus.go

```diff
@@ -121,7 +150,7 @@ func (cs *ContainerStatus) UnmarshalJSON(b []byte) error {
 	}
 	if b[0] != '"' || b[len(b)-1] != '"' {
 		*cs = ContainerStatusNone
-		return errors.New("ContainerStatus must be a string or null; Got " + string(b))
+		return errors.New("containerStatus must be a string or null; Got " + string(b))
```

Comment:
Can you reformat this as: `"container status unmarshal: status must be a string or null.."`